packages/components/timezone/timezone-option-tokens.scss

```
@@ -0,0 +1,51 @@
+.kbq-timezone-option {
```

Думаю, лучше выбрать такой селектор, чтобы селектор не пробегал по каждому `timezone-option`, а остановился только на панели `timezone-select`

```suggestion
.kbq-timezone-select__panel {
```